lib/active_analytics.rb

```diff
@@ -18,15 +20,36 @@ def self.redis
     @redis ||= Redis.new(url: redis_url)
   end
 
-  def self.record_request(request)
-    params = {
-      site: request.host,
-      page: request.path,
-      date: Date.today,
+  def self.user_agent_extractors
+    @@user_agent_extractors ||= {
+      browser: lambda do |b|
```

Comment:
Is there a limitation to use browser.name, browser.device.name and browser.platform.name ?

Reply:
No, this was just me wanting to make 100% sure I know all the possible values that could be written. Just as you mentioned (https://github.com/BaseSecrete/active_analytics/issues/6#issuecomment-1971452984): The potential for far too many rows is big.